pyvista/core/utilities/geometric_sources.py

```diff
@@ -3384,3 +3386,192 @@ def _make_axes_parts(
+            valid_sign = [sign] * 3
+        else:
+            _validation.check_length(sign, exact_length=3)
+            [_check_sign(s) for s in sign]
+            valid_sign = sign
+        self._normal_sign = tuple(valid_sign)
+
+    @property
+    def resolution(self) -> tuple[int, int, int]:  # numpydoc ignore=RT01
+        """Return or set the resolution of the planes."""
+        return self._resolution
+
+    @resolution.setter
+    def resolution(self, resolution: int | VectorLike[int]):  # numpydoc ignore=GL08
+        self._resolution = _validation.validate_array3(
+            resolution, broadcast=True, to_tuple=True, name='resolution'
+        )
+
+    @property
+    def bounds(self) -> BoundsLike:  # numpydoc ignore=RT01
+        """Return or set the bounds of the planes."""
+        return self._bounds
+
+    @bounds.setter
+    def bounds(self, bounds: BoundsLike):  # numpydoc ignore=GL08
+        self._bounds = _validation.validate_array(
+            bounds, dtype_out=float, must_have_length=6, to_tuple=True, name='bounds'
+        )
+
+    @property
+    def names(self) -> tuple[str, str, str]:  # numpydoc ignore=RT01
+        """Return or set the names of the planes."""
+        return self._names
+
+    @names.setter
+    def names(self, names: Sequence[str]):  # numpydoc ignore=GL08
+        _validation.check_instance(names, (tuple, list), name='names')
+        _validation.check_iterable_items(names, str, name='names')
+        _validation.check_length(names, exact_length=3, name='names')
+        self._names = cast(Tuple[str, str, str], tuple(names))
+
+    def update(self):
+        """Update the output of the source."""
+        ORIGIN = (0.0, 0.0, 0.0)
+
+        # Unpack vars
+        xy_source, yz_source, zx_source = self._plane_sources
+        x_res, y_res, z_res = self.resolution
+        x_min, x_max, y_min, y_max, z_min, z_max = self.bounds
+
+        # Compute bounds-related vars
+        x_size, y_size, z_size = x_max - x_min, y_max - y_min, z_max - z_min
+        center = (x_max + x_min) / 2, (y_max + y_min) / 2, (z_max + z_min) / 2
+
+        # Update plane sources
+        xy_source.SetXResolution(x_res)
+        xy_source.SetYResolution(y_res)
+        xy_source.SetPoint1(x_size, 0.0, 0.0)
+        xy_source.SetPoint2(0.0, y_size, 0.0)
+        xy_source.SetOrigin(ORIGIN)
+        xy_source.SetCenter(center)
+        xy_source.Update()
+
+        yz_source.SetXResolution(y_res)
+        yz_source.SetYResolution(z_res)
+        yz_source.SetPoint1(0.0, y_size, 0.0)
+        yz_source.SetPoint2(0.0, 0.0, z_size)
+        yz_source.SetOrigin(ORIGIN)
+        yz_source.SetCenter(center)
+        yz_source.Update()
+
+        zx_source.SetXResolution(z_res)
+        zx_source.SetYResolution(x_res)
+        zx_source.SetPoint1(0.0, 0.0, z_size)
+        zx_source.SetPoint2(x_size, 0.0, 0.0)
+        zx_source.SetOrigin(ORIGIN)
+        zx_source.SetCenter(center)
+        zx_source.Update()
```

Comment:
I plan to update this after merging #6372.

Reply:
Thanks!

I think having an explicit `update` method is good to have too, otherwise we need to get the `output` property without actually storing its value to implicitly do the update.

``` python
zx_source.output
```

which `ruff` will complain about:
```
B018 Found useless expression. Either assign it to a variable or remove it.
```

so then we need to do this instead which isn't a great solution IMO 
```python
_ = zx_source.output
```

Reply:
@tkoyama010 if you're OK with it I think we can update this PR with changes from https://github.com/pyvista/pyvista/pull/6372, but keep the existing `Update()` and `copy_from(GetOutput())` mechanics for now, as I think those should probably be batch-updated for all sources that use `pv.wrap` (with tests for the whole module  to verify that `pv_source.output is pv_source.output` is always `True`), e.g.:
``` diff
-        xy_source.SetXResolution(x_res)
-        xy_source.SetYResolution(y_res)
-        xy_source.SetPoint1(x_size, 0.0, 0.0)
-        xy_source.SetPoint2(0.0, y_size, 0.0)
-        xy_source.SetOrigin(ORIGIN)
-        xy_source.SetCenter(center)
+        xy_source.i_resolution = x_res
+        xy_source.j_resolution = y_res
+        xy_source.point_a = x_size, 0.0, 0.0
+        xy_source.point_b = 0.0, y_size, 0.0
+        xy_source.origin = ORIGIN
+        xy_source.center = center
         xy_source.Update()
```

Reply:
Resolved by https://github.com/pyvista/pyvista/pull/6366/commits/0854c1c4d512ac99749abe0f6ff2e34be68e08ac